src/lib/Microsoft.Health.Fhir.R4.Ingest/Service/R4DeviceAndPatientCreateIdentityService.cs

```diff
@@ -10,10 +10,13 @@
 using Microsoft.Health.Extensions.Fhir;
 using Microsoft.Health.Extensions.Fhir.Service;
 using Microsoft.Health.Fhir.Ingest.Data;
+using Microsoft.Health.Fhir.Ingest.Host;
 using Model = Hl7.Fhir.Model;
 
 namespace Microsoft.Health.Fhir.Ingest.Service
 {
+    [ResourceIdentityService("Create")]

```

Comment:
If you add optional enumeration ctor then you can update to [ResourceIdentityService(ResourceIdentityServiceType.Create)]

Reply:
Updated.